lib/matplotlib/textpath.py

```diff
@@ -400,6 +400,8 @@ def __init__(self, xy, s, size=None, prop=None,
 
         if prop is None:
             prop = FontProperties()
+        else:
+            prop = FontProperties._from_any(prop)
```

Comment:
Did you forget to copy result, like you have done above? Maybe `_from_any` should return copy always?

Reply:
For now I decided to keep the previous (probably bad) semantics where someone could e.g. pass in a FontProperties instance and, depending on how the thing was written, changes to the *original* instance may or may not affect the downstream artist.  I agree always making a copy is likely to result in more consistent behavior, but let's not change that here.